packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```diff
@@ -109,6 +109,7 @@ install -m 644 .%{gem_instdir}/config/cli_config.template.yml \
 %files
 %dir %{gem_instdir}
 %{_root_bindir}/hammer
+%{_root_bindir}/hammer-complete
```

Comment:
A side effect of this is that `ha<tab>` wil mean the user needs to type a space and then tab again. I'd recommend to look at options to avoid this. A common way to deal with this is using libexec.